### PR TITLE
Client meeting agendas

### PR DESCRIPTION
**Pull request by mkuelker**
Add agendas to main

### CODE REVIEW

**Comment by LoganParker** [comment]
should we wait till meeting notes are taken before merging? Idk otherwise looks great, thanks for making the agenda @mkuelker !

File: Agendas/23_01_27.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# Schedule
+*Teams will likely go in sequential order. Meeting shouldn't be longer than 30 minutes*
+
+# TEAM A Items
+ - Demo API and updated charactar animations
+ 
+
+# Questions
+- Processing times and recognizers. What should be the default option. (Preferably include both.) (Show the animations on discord)
```
Comment (LoganParker) [inline Agendas/23_01_27.md:12]: Don't forget to bring up the differences between the English processor and non English processor on English audio! We want to find out if we can ignore the pocket sphinx recognizer entirely and rely on the other one

**Review by LoganParker — APPROVED** [review]
Thank you for prepping this! It is much appreciated :)

**Review by edouarde1 — APPROVED** [review]
Thanks for the agenda, I will make sure to minutes ready before the meeting

**Pull request merged**